app/services/activity_defaults.rb

```diff
@@ -35,6 +36,10 @@ def is_oda?
     @is_oda.nil? ? @parent_activity.is_oda : @is_oda
   end
 
+  def publish_to_iati?
```

Comment:
Is this needed. Can we just default `publish_to_iati` to true, then set it to false in `set_non_oda_defaults`?

Reply:
Good question - I wasn't sure whether `is_oda_changed?` would be true when creating projects/third-party projects that inherit `is_oda` during instantiation, and when creating activities via bulk upload. Does `_changed?` return `true` on creation or just when something is changed from one value to another?

Can easily test it by removing this and seeing if the tests pass!

Reply:
I think you're right - the only tests that fail when I comment out this stuff are in the `ActivityDefaults` spec, and the values are tested elsewhere